src/backend/metal/src/command.rs

```diff
@@ -24,7 +24,6 @@ use cocoa::foundation::{NSUInteger, NSInteger, NSRange};
 use block::{ConcreteBlock};
 use smallvec::SmallVec;
 
-
```

Comment:
fwiw, I like leaving 2 empty lines after the `use` section to separate "headers" from "body"